pkg/serverstate/statetest/test_pipeline.go

```diff
@@ -422,4 +422,103 @@ func TestPipeline(t *testing.T, factory Factory, restartF RestartFactory) {
 			require.Len(resp, 3)
 		}
 	})
+
+	t.Run("Step Workspace", func(t *testing.T) {
+		require := require.New(t)
+
+		s := factory(t)
+		defer s.Close()
+
+		// Set
+		p := ptypes.TestPipeline(t, nil)
+		err := s.PipelinePut(p)
+		require.NoError(err) // no job id set
```

Comment:
```suggestion
		require.NoError(err)
```

That comment might of been copy/pasted